provider/core_security_list_resource_test.go

```diff
@@ -35,16 +31,198 @@ func (s *ResourceCoreSecurityListTestSuite) SetupTest() {
 			cidr_block = "10.0.0.0/16"
 			compartment_id = "${var.compartment_id}"
 			display_name = "-tf-vcn"
+		}
+	    data "oci_core_security_lists" "t" {
```

Comment:
nit: indentation looks off.

Reply:
I did a bunch of refactoring in this file in the latest revision in order to merge with the default resource changes... please take another look.